### ISSUE TITLE
You can remove default_app_config

### ISSUE THREAD

**Issue by bittner** [penn-state-dance-marathon/python3-saml-django#15]
Since at least Django 3.2 Django discourages the declaration of `default_app_config`.

```python
.../python3.9/site-packages/django/apps/registry.py:91: RemovedInDjango41Warning: 'django_saml' defines
default_app_config = 'django_saml.apps.DjangoSamlConfig'. Django now detects this configuration automatically.
You can remove default_app_config.
```

According to that warning, `default_app_config` can be removed [from `django_saml/__init__.py`](https://github.com/penn-state-dance-marathon/python3-saml-django/blob/master/django_saml/__init__.py#L1).

**Comment by ngearhart** [comment]
Good call. You are welcome to open a pull request to fix this, otherwise it will be fixed in the coming weeks.

**Comment by bittner** [comment]
This is a "won't fix" as long as Django 2.2 needs to be supported.

Sorry for the noise!
